diff -Nur glade3-3.3.1/configure.ac glade3-3.3.1.new/configure.ac
--- glade3-3.3.1/configure.ac	2007-06-18 17:12:38.000000000 +0200
+++ glade3-3.3.1.new/configure.ac	2007-06-19 11:17:41.000000000 +0200
@@ -87,7 +87,7 @@
 dnl ================================================================
 dnl Check for gtk+
 dnl ================================================================
-PKG_CHECK_MODULES(GTK, [gtk+-2.0 >= 2.10.0  libxml-2.0 >= 2.4.0])
+PKG_CHECK_MODULES(GTK, [gtk+-2.0 >= 2.10.0  libxml-2.0 >= 2.4.0 launchpad-integration])
 AC_SUBST(GTK_LIBS)
 AC_SUBST(GTK_CFLAGS)
 
diff -Nur glade3-3.3.1/src/glade-project-window.c glade3-3.3.1.new/src/glade-project-window.c
--- glade3-3.3.1/src/glade-project-window.c	2007-06-18 15:34:43.000000000 +0200
+++ glade3-3.3.1.new/src/glade-project-window.c	2007-06-19 11:17:41.000000000 +0200
@@ -36,6 +36,7 @@
 #include <glib/gi18n.h>
 #include <gdk/gdkkeysyms.h>
 #include <gtk/gtkstock.h>
+#include <launchpad-integration.h>
 
 #define GLADE_ACTION_GROUP_STATIC "GladeStatic"
 #define GLADE_ACTION_GROUP_PROJECT "GladeProject"
@@ -1779,6 +1780,8 @@
 "      <menuitem action='HelpContents'/>\n"
 "      <menuitem action='DeveloperReference'/>\n"
 "      <separator/>\n"
+"      <placeholder name='LaunchpadIntegrationPlaceholder'/>\n"
+"      <separator/>\n"
 "      <menuitem action='About'/>\n"
 "    </menu>\n"
 "  </menubar>\n"
@@ -1964,6 +1967,7 @@
 gpw_construct_menu (GladeProjectWindow *gpw)
 {
 	GError *error = NULL;
+	GtkActionGroup *group;
 	
 	gpw->priv->static_actions = gtk_action_group_new (GLADE_ACTION_GROUP_STATIC);
 	gtk_action_group_set_translation_domain (gpw->priv->static_actions, GETTEXT_PACKAGE);
@@ -2004,6 +2008,10 @@
 	gtk_ui_manager_insert_action_group (gpw->priv->ui, gpw->priv->project_actions, 1);
 	gtk_ui_manager_insert_action_group (gpw->priv->ui, gpw->priv->projects_list_menu_actions, 3);
 	
+	group = launchpad_integration_get_action_group ();
+	gtk_ui_manager_insert_action_group (gpw->priv->ui, group, 4);
+	g_object_unref (group);		
+	
 	gtk_window_add_accel_group (GTK_WINDOW (gpw->priv->window), 
 				  gtk_ui_manager_get_accel_group (gpw->priv->ui));
 
@@ -2013,6 +2021,8 @@
 		g_error_free (error);
 	}
 	
+	launchpad_integration_add_ui (gpw->priv->ui, "/MenuBar/HelpMenu/LaunchpadIntegrationPlaceholder");
+	
 	return gtk_ui_manager_get_widget (gpw->priv->ui, "/MenuBar");
 }
 
@@ -2489,6 +2499,8 @@
 	gpw->priv->label = NULL;
 	gpw->priv->default_path = NULL;
 	
+	launchpad_integration_set_sourcepackagename ("glade-3");	
+	
 	gtk_about_dialog_set_url_hook ((GtkAboutDialogActivateLinkFunc) about_dialog_activate_link_func, gpw, NULL);
 	
 }
