Index: gladeui/glade-base-editor.c
===================================================================
--- gladeui/glade-base-editor.c	(revision 1826)
+++ gladeui/glade-base-editor.c	(working copy)
@@ -377,7 +377,7 @@
 glade_base_editor_update_properties_idle (gpointer data)
 {
 	GladeBaseEditor *editor = (GladeBaseEditor *)data;
-	glade_base_editor_treeview_cursor_changed (NULL, editor);
+	glade_base_editor_treeview_cursor_changed (editor->priv->treeview, editor);
 	editor->priv->properties_idle = 0;
 	return FALSE;
 }
@@ -1146,7 +1146,7 @@
 	}
 
 	/* Set the name */
-	glade_widget_set_name (gchild_new, name);
+	glade_command_set_name (gchild_new, name);
 	
 	if (GTK_IS_WIDGET (child_new))
 		gtk_widget_show_all (GTK_WIDGET (child_new));
Index: gladeui/glade-project.c
===================================================================
--- gladeui/glade-project.c	(revision 1826)
+++ gladeui/glade-project.c	(working copy)
@@ -1915,6 +1915,7 @@
 
 	g_return_if_fail (GLADE_IS_PROJECT (project));
 	g_return_if_fail (GLADE_IS_WIDGET (glade_widget));
+	g_return_if_fail (widget_name);
 
 	do
 	{
