Index: glade3-3.8.5/gladeui/glade-widget.c
===================================================================
--- glade3-3.8.5.orig/gladeui/glade-widget.c
+++ glade3-3.8.5/gladeui/glade-widget.c
@@ -1928,6 +1928,26 @@ _glade_widget_peek_prop_refs (GladeWidge
   return widget->prop_refs;
 }
 
+/* Tells Ubuntu to show any embedded menu bars */
+static void
+ubuntu_show_menu_bars (GtkWidget *widget)
+{
+	if (GTK_IS_MENU_BAR (widget) &&
+	    g_object_class_find_property (G_OBJECT_GET_CLASS (widget),
+	                                  "ubuntu-local") != NULL)
+	{
+		g_object_set (G_OBJECT (widget),
+		              "ubuntu-local", TRUE,
+		              NULL);
+	}
+	else if (GTK_IS_CONTAINER (widget))
+	{
+		gtk_container_foreach (GTK_CONTAINER (widget),
+		                       (GtkCallback)ubuntu_show_menu_bars,
+		                       NULL);
+	}
+}
+
 /*******************************************************************************
                                      API
  *******************************************************************************/
@@ -4486,6 +4506,8 @@ glade_widget_embed (GladeWidget *gwidget
 	
 	if (glade_window_is_embedded (window)) return TRUE;
 	
+	ubuntu_show_menu_bars (widget);
+
 	if (gtk_widget_get_realized (widget)) gtk_widget_unrealize (widget);
 
 	GTK_WIDGET_UNSET_FLAGS (widget, GTK_TOPLEVEL);
