Description: link required gmodule
Author: Julian Taylor <jtaylor@ubuntu.com>
Applied-Upstream: 3.12.0

--- a/configure.ac
+++ b/configure.ac
@@ -121,7 +121,7 @@
 dnl ================================================================
 dnl Check for gtk+
 dnl ================================================================
-PKG_CHECK_MODULES(GTK, [gtk+-2.0 >= 2.20 gthread-2.0 libxml-2.0 >= 2.4.0])
+PKG_CHECK_MODULES(GTK, [gtk+-2.0 >= 2.20 gthread-2.0 gmodule-2.0 libxml-2.0 >= 2.4.0])
 AC_SUBST(GTK_LIBS)
 AC_SUBST(GTK_CFLAGS)
 
